go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/lookfirst-io/go-openai
 
 go 1.18
+
+require github.com/sashabaranov/go-openai v1.26.0 // indirect
```

Comment:
Why is this now being imported? I can't see it being used

Please run `go mod tidy`

Reply:
done